src/controllers/playback/video/index.js

```diff
@@ -1384,11 +1384,13 @@ export default function (view) {
         let chapterThumbContainer = bubble.querySelector('.chapterThumbContainer');
         let chapterThumb;
         let chapterThumbText;
+        let chapterThumbName;
 
         // Create bubble elements if they don't already exist
         if (chapterThumbContainer) {
             chapterThumb = chapterThumbContainer.querySelector('.chapterThumbWrapper');
             chapterThumbText = chapterThumbContainer.querySelector('.chapterThumbText');
+            chapterThumbName = chapterThumbContainer.querySelector('.chapterThumbName');
```

Comment:
```suggestion
            chapterThumbText = chapterThumbContainer.querySelector('h2.chapterThumbText');
            chapterThumbName = chapterThumbContainer.querySelector('div.chapterThumbText');
```
Here we probably need to introduce `chapterThumbTime` and (_as you did_) `chapterThumbName`.